back/engines/free/email_campaigns/config/locales/pl-PL.yml

```
@@ -33,7 +33,7 @@ pl:
       "official_feedback_on_your_initiative": Aktualizacja mojej propozycji
       "password_reset": Zmiana hasła
       "project_moderation_rights_received": Przyznanie prawa do moderacji projektu
-      "project_folder_moderation_rights_received": Folder manager rights received
+      "project_folder_moderation_rights_received": Przydzielono uprawnienia menedżera folderów
```

Maybe these strings need to be wrapped in single quotes? 
```suggestion
      "project_folder_moderation_rights_received": 'Przydzielono uprawnienia menedżera folderów'
```